src/cli/abrtcli/cli/__init__.py

```diff
@@ -0,0 +1,113 @@
+from argparse import Action
+from glob import glob
+from importlib import import_module
+from os.path import basename, dirname
+
+from abrtcli import config
+from abrtcli.i18n import _
+from reportclient import set_verbosity
+
+class Command:
+    aliases = []
+
+    def __init__(self, subparsers):
+        self._parser = subparsers.add_parser(self.name, aliases=self.aliases,
+                help=self.description, description=self.description)
```

Comment:
No-member `name`, `description`.

Reply:
Hmm, this was more or less intended, but probably not a good idea.